### PR TITLE
#171375416 Fixes get user trip requests

### PR DESCRIPTION
**Pull request by VedaDom**
#### What does this PR do?
Fixes get user trip requests with all related information

#### What are the relevant pivotal tracker stories?
- [#171375416](https://www.pivotaltracker.com/story/show/171375416)

#### Screenshots (if appropriate)
Screen Shot 2020-02-20 at 15 37 03

### CODE REVIEW

File: src/controllers/trip.controller.js [hunk 1]
```diff
@@ -124,24 +124,20 @@ class TripController {
...
-        res,
-        'My Trip Requests',
-        200,
-        requests
-      );
-    }
-    return response.errorMessage(
+    const manager = await db.user.findOne({ where: requests.managerId });
+    const requestTrips = await tripRequestService.getTripRequestsOfUser(requests, manager);
+    return response.successMessage(
```
Comment (bdushimi) [inline src/controllers/trip.controller.js:132]: @VedaDom what if there are not trip requests for a particular user? That's why there was an if condition though the response status code returned was not correct (line 141)!

File: src/tests/getUserRequests.test.js [hunk 2]
```diff
@@ -109,18 +109,6 @@ describe('/trip/myrequests', () => {
         done();
       });
   });
```
Comment (bdushimi) [inline src/tests/getUserRequests.test.js:111]: @VedaDom any reason why this test case is omitted?

**Review by bdushimi — CHANGES_REQUESTED** [review]

File: src/tests/getUserRequests.test.js (hunk 2 above)
Comment (VedaDom) [reply src/tests/getUserRequests.test.js:111]: The reason why I removed is that there was no response with the expected results.

File: src/controllers/trip.controller.js (hunk 1 above)
Comment (VedaDom) [reply src/controllers/trip.controller.js:132]: I  want to return empty array when no records found.

**Review by bdushimi — APPROVED** [review]

**Pull request merged**